mautic/mautic
# Fix MJML issues with Brienz template

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ✅ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

Note: Making this PR on behalf of @jos0405 as he had some Git problems!

This PR:

- fixes issues with the Brienz template
- Added logo to the top
- Fully mjml compatible

Litmus test:
[https://litmus.com/pub/a9815cd](https://litmus.com/pub/a9815cd)

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new email, choose Brienz template
3. Check if text is not losing format once you edit the text.

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11356"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
